src/app/(fullscreen)/verify/email/page.tsx

```
@@ -75,40 +78,57 @@ export default function EmailStep() {
+              </motion.div>
+            </button>
+            <AnimatePresence>
+              {dropdownOpen && (
+                <motion.ul
+                  className="absolute z-10 mt-1.5 w-full overflow-hidden rounded-md bg-gray-900"
+                  role="listbox"
+                  tabIndex={-1}
+                  variants={dropdownVariants}
+                  initial="hidden"
+                  animate="visible"
+                  exit="exit"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)